rust/rules.bzl

```diff
@@ -0,0 +1,117 @@
+load("@io_bazel_rules_rust//rust:rust.bzl", "rust_library")
+load("//protobuf:rules.bzl", "proto_compile", "proto_language_deps", "proto_repositories")
+load("//rust/raze:crates.bzl", "raze_fetch_remote_crates")
+
+def rust_proto_repositories(
+    lang_deps = {},
+    lang_requires = [],
+    **kwargs):
+  raze_fetch_remote_crates()
+
+def rust_proto_compile(langs = [str(Label("//rust"))], **kwargs):
+  proto_compile(langs = langs, **kwargs)
+
+PB_COMPILE_DEPS = [
+    "@org_pubref_rules_protobuf//rust/raze:protobuf",
+]
+
+GRPC_COMPILE_DEPS = PB_COMPILE_DEPS + [
```

Comment:
I think these need to be a (default) parameter of the `rust_proto_library` rule. My thinking is that folks will want to centrally control the whole set of third party crates that they bring in, and maintaining compatibility with the dependencies that `rust_proto_library` uses might be important.

In the event that they don't care (yet), the default value will give them something functional.

Reply:
Add a compile_deps argument, defaulted to None so people can overwrite it.